project/frontend/src/index.js

```diff
@@ -13,18 +13,27 @@ import MapContainer from './components/ResultsMap.js';
 import MatchResults from './pages/MatchResults.js';
 import PreferenceForm from './components/PreferenceForm.js';
 
+import RestaurantCard from './components/RestaurantCard.js';
+
 import * as serviceWorker from './serviceWorker';
 
+// TODO: remove
+import { restaurant, details } from './components/SampleRestaurant.js';
+
 ReactDOM.render(
   <React.StrictMode>
     <Authentication>
       <Router>
         <Header />
         <Switch>
-          <Route exact path='/' component={HomePage} />
+          <Route path='/card'>
+            <RestaurantCard restaurant={restaurant} details={details} />
```

Comment:
Do we not need component={RestaurantCard}?

Reply:
No, you can either specify the child in `component=` or just give the `Route` explicit children. I needed to give props to `RestaurantCard` so I went the descendant route instead of `component=`.